src/masternodes/mn_checks.cpp

```diff
@@ -2171,6 +2180,164 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor {
         return Res::Ok();
     }
 
+    
+    Res operator()(const CDUSDLockMessage& obj) const {
+        auto res = CheckCustomTx();
+        if (!res)
+            return res;
+
+        if (!HasAuth(obj.owner)) {
+            return Res::Err("Transaction must have at least one input from owner");
+        }
+
+        const auto attributes = mnview.GetAttributes();
+        if (!attributes) {
```

Comment:
this can be only `assert(attributes)` as attributes exist already and if not found then it is a critical error.